openapi/api.github.com/operations/repos/get.json

```diff
@@ -1,21 +1,22 @@
 {
-  "summary": "Get",
-  "description": "The `parent` and `source` objects are present when the repository is a fork. `parent` is the repository this repository was forked from, `source` is the ultimate source for the network.",
+  "summary": "Get a repository",
+  "description": "When you pass the `scarlet-witch-preview` media type, requests to get a repository will also return the repository's code of conduct if it can be detected from the repository's code of conduct file.\n\nThe `parent` and `source` objects are present when the repository is a fork. `parent` is the repository this repository was forked from, `source` is the ultimate source for the network.",
   "operationId": "repos/get",
   "tags": ["repos"],
   "externalDocs": {
     "description": "API method documentation",
-    "url": "https://developer.github.com/v3/repos/#get"
+    "url": "https://developer.github.com/v3/repos/#get-a-repository"
   },
   "parameters": [
     {
       "name": "accept",
-      "description": "Setting to `application/vnd.github.v3+json` is recommended",
+      "description": "This API is under preview and subject to change.",
       "in": "header",
       "schema": {
         "type": "string",
-        "default": "application/vnd.github.v3+json"
-      }
+        "default": "application/vnd.github.scarlet-witch-preview+json"
+      },
+      "required": true
```

Comment:
Incorrect.

Reply:
will be addressed with a follow up PR, won't cause much harm to merge it in, I don't want these changes to be blocked for too long